lib/semian/rails.rb

```diff
@@ -0,0 +1,7 @@
+require 'active_record/connection_adapters/abstract_adapter'
+
+class ActiveRecord::ConnectionAdapters::AbstractAdapter
```

Comment:
No because if you are in the context of a Rails app, the `Bundler.require` is triggered by Rails itself early it's boot process, so effectively when semian is required ActiveSupport should be there.